File: CS/CategoryWrapper.cs
namespace PreviewClickDemo {
    public class CategoryWrapper {
        public bool IsExpanded { get; set; }
        public Northwind.CategoriesRow Row { get; private set; }

        public CategoryWrapper(bool isExpanded, Northwind.CategoriesRow row) {
            IsExpanded = isExpanded;
            Row = row;
        }
    }
}

File: VB/CategoryWrapper.vb
Namespace PreviewClickDemo
    Public Class CategoryWrapper
        Public Property IsExpanded() As Boolean
        Private privateRow As Northwind.CategoriesRow
        Public Property Row() As Northwind.CategoriesRow
            Get
                Return privateRow
            End Get
            Private Set(ByVal value As Northwind.CategoriesRow)
                privateRow = value
            End Set
        End Property

        Public Sub New(ByVal isExpanded As Boolean, ByVal row As Northwind.CategoriesRow)
            Me.IsExpanded = isExpanded
            Me.Row = row
        End Sub
    End Class
End Namespace
